firmware/arduino/chessboard/chessboard.ino

```diff
@@ -97,9 +96,8 @@ void setup()
   pinMode(Y_AXIS_ENDSTOP_SWITCH, INPUT);
   pinMode(CHESS_TIMER_BUTTON, INPUT);
 
-  // Defines different units of board being used
+  // Defines units of board being used
```

Comment:
This is just my preference, but I like to avoid using the format "A of B" and just use "B's A"
In this case, it would be "the board's units" instead of "units of board"